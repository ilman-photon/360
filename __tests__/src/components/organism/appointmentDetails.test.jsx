import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import AppointmentDetails from "../../../../src/components/organisms/ScheduleAppointment/appointmentDetails";

describe("App", () => {
    let container;
    beforeEach(() => {
      container = render(<AppointmentDetails />);
    });
  
    it("AppointmentDetails render", () => {
        expect(container).toMatchSnapshot();
    });
    
  });
  
