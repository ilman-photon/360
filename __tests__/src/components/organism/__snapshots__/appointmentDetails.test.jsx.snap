// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App AppointmentDetails render 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="MuiBox-root css-0"
      >
        <div
          class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root card css-1v5z4dq-MuiPaper-root-MuiCard-root"
        >
          <div
            class="MuiCardHeader-root cardHeader css-1ele1k2-MuiCardHeader-root"
          >
            <div
              class="MuiCardHeader-content css-1qbkelo-MuiCardHeader-content"
            >
              <span
                class="MuiTypography-root MuiTypography-h5 MuiCardHeader-title css-1qvr50w-MuiTypography-root"
              >
                <div>
                  <span
                    style="margin-right: 8px; display: flex; align-items: center;"
                  />
                  <div
                    aria-label="appointmentDetails heading"
                    class="title"
                    style="font-weight: 700;"
                  >
                    appointmentDetails
                  </div>
                </div>
              </span>
            </div>
            <div
              class="MuiCardHeader-action css-sgoict-MuiCardHeader-action"
            >
              <button
                class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root  css-1e6y48t-MuiButtonBase-root-MuiButton-root"
                tabindex="0"
                type="button"
              >
                <svg
                  aria-hidden="false"
                  aria-label="Edit icon"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1x727db-MuiSvgIcon-root"
                  data-testid="EditOutlinedIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="m14.06 9.02.92.92L5.92 19H5v-.92l9.06-9.06M17.66 3c-.25 0-.51.1-.7.29l-1.83 1.83 3.75 3.75 1.83-1.83c.39-.39.39-1.02 0-1.41l-2.34-2.34c-.2-.2-.45-.29-.71-.29zm-3.6 3.19L3 17.25V21h3.75L17.81 9.94l-3.75-3.75z"
                  />
                </svg>
                <div
                  style="margin-left: 4px; color: rgb(0, 130, 148); text-decoration-line: underline; text-transform: capitalize; font-size: 18px;"
                  type="link"
                >
                  Edit
                </div>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </div>
          </div>
          <div
            class="MuiCardContent-root css-vp7ejv-MuiCardContent-root"
          >
            <div
              class="css-1p5q5e5-MuiStack-root"
            >
              <div
                class="css-nen11g-MuiStack-root"
              >
                <div
                  class="labelContainer"
                  style="justify-content: unset;"
                >
                  <span
                    style="margin-right: 8px; display: flex; align-items: center; width: 18px;"
                  >
                    <svg
                      aria-hidden="false"
                      aria-label="Calendar icon"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                      data-testid="CalendarTodayIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 3h-1V1h-2v2H7V1H5v2H4c-1.1 0-2 .9-2 2v16c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm0 18H4V8h16v13z"
                      />
                    </svg>
                  </span>
                  <div
                    class="label"
                    style="padding-left: 4px; color: rgb(0, 59, 74);"
                  >
                    Date and time
                  </div>
                </div>
                <div
                  class="css-1okj3ks-MuiStack-root"
                >
                  <div
                    style="margin-top: 10px;"
                  >
                    <span
                      class="MuiTypography-root MuiTypography-bodyMedium css-1oitrp4-MuiTypography-root"
                    >
                      Tuesday, Sep 06 - 10:37 pm
                    </span>
                  </div>
                </div>
              </div>
              <div
                class="css-nen11g-MuiStack-root"
              >
                <div
                  class="labelContainer"
                  style="justify-content: unset;"
                >
                  <span
                    style="margin-right: 8px; display: flex; align-items: center; width: 18px;"
                  >
                    <svg
                      aria-hidden="false"
                      aria-label="Mail icon"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                      data-testid="ContactMailIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M21 8V7l-3 2-3-2v1l3 2 3-2zm1-5H2C.9 3 0 3.9 0 5v14c0 1.1.9 2 2 2h20c1.1 0 1.99-.9 1.99-2L24 5c0-1.1-.9-2-2-2zM8 6c1.66 0 3 1.34 3 3s-1.34 3-3 3-3-1.34-3-3 1.34-3 3-3zm6 12H2v-1c0-2 4-3.1 6-3.1s6 1.1 6 3.1v1zm8-6h-8V6h8v6z"
                      />
                    </svg>
                  </span>
                  <div
                    class="label"
                    style="padding-left: 4px; color: rgb(0, 59, 74);"
                  >
                    Insurance
                  </div>
                </div>
                <div
                  class="css-1okj3ks-MuiStack-root"
                >
                  <div
                    style="margin-top: 10px;"
                  >
                    <span
                      class="MuiTypography-root MuiTypography-bodyMedium css-1oitrp4-MuiTypography-root"
                    >
                      No Insurance provided
                    </span>
                  </div>
                </div>
              </div>
              <div
                class="css-nen11g-MuiStack-root"
              >
                <div
                  class="labelContainer"
                  style="justify-content: unset;"
                >
                  <span
                    style="margin-right: 8px; display: flex; align-items: center; width: 18px;"
                  >
                    <svg
                      aria-hidden="false"
                      aria-label="Business icon"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                      data-testid="BusinessIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 7V3H2v18h20V7H12zM6 19H4v-2h2v2zm0-4H4v-2h2v2zm0-4H4V9h2v2zm0-4H4V5h2v2zm4 12H8v-2h2v2zm0-4H8v-2h2v2zm0-4H8V9h2v2zm0-4H8V5h2v2zm10 12h-8v-2h2v-2h-2v-2h2v-2h-2V9h8v10zm-2-8h-2v2h2v-2zm0 4h-2v2h2v-2z"
                      />
                    </svg>
                  </span>
                  <div
                    class="label"
                    style="padding-left: 4px; color: rgb(0, 59, 74);"
                  >
                    Purpose of visit
                  </div>
                </div>
                <div
                  class="css-1okj3ks-MuiStack-root"
                >
                  <div
                    style="margin-top: 10px;"
                  >
                    <span
                      class="MuiTypography-root MuiTypography-bodyMedium css-1oitrp4-MuiTypography-root"
                    >
                      -
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="MuiBox-root css-0"
    >
      <div
        class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root card css-1v5z4dq-MuiPaper-root-MuiCard-root"
      >
        <div
          class="MuiCardHeader-root cardHeader css-1ele1k2-MuiCardHeader-root"
        >
          <div
            class="MuiCardHeader-content css-1qbkelo-MuiCardHeader-content"
          >
            <span
              class="MuiTypography-root MuiTypography-h5 MuiCardHeader-title css-1qvr50w-MuiTypography-root"
            >
              <div>
                <span
                  style="margin-right: 8px; display: flex; align-items: center;"
                />
                <div
                  aria-label="appointmentDetails heading"
                  class="title"
                  style="font-weight: 700;"
                >
                  appointmentDetails
                </div>
              </div>
            </span>
          </div>
          <div
            class="MuiCardHeader-action css-sgoict-MuiCardHeader-action"
          >
            <button
              class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root  css-1e6y48t-MuiButtonBase-root-MuiButton-root"
              tabindex="0"
              type="button"
            >
              <svg
                aria-hidden="false"
                aria-label="Edit icon"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1x727db-MuiSvgIcon-root"
                data-testid="EditOutlinedIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="m14.06 9.02.92.92L5.92 19H5v-.92l9.06-9.06M17.66 3c-.25 0-.51.1-.7.29l-1.83 1.83 3.75 3.75 1.83-1.83c.39-.39.39-1.02 0-1.41l-2.34-2.34c-.2-.2-.45-.29-.71-.29zm-3.6 3.19L3 17.25V21h3.75L17.81 9.94l-3.75-3.75z"
                />
              </svg>
              <div
                style="margin-left: 4px; color: rgb(0, 130, 148); text-decoration-line: underline; text-transform: capitalize; font-size: 18px;"
                type="link"
              >
                Edit
              </div>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
        <div
          class="MuiCardContent-root css-vp7ejv-MuiCardContent-root"
        >
          <div
            class="css-1p5q5e5-MuiStack-root"
          >
            <div
              class="css-nen11g-MuiStack-root"
            >
              <div
                class="labelContainer"
                style="justify-content: unset;"
              >
                <span
                  style="margin-right: 8px; display: flex; align-items: center; width: 18px;"
                >
                  <svg
                    aria-hidden="false"
                    aria-label="Calendar icon"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                    data-testid="CalendarTodayIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M20 3h-1V1h-2v2H7V1H5v2H4c-1.1 0-2 .9-2 2v16c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm0 18H4V8h16v13z"
                    />
                  </svg>
                </span>
                <div
                  class="label"
                  style="padding-left: 4px; color: rgb(0, 59, 74);"
                >
                  Date and time
                </div>
              </div>
              <div
                class="css-1okj3ks-MuiStack-root"
              >
                <div
                  style="margin-top: 10px;"
                >
                  <span
                    class="MuiTypography-root MuiTypography-bodyMedium css-1oitrp4-MuiTypography-root"
                  >
                    Tuesday, Sep 06 - 10:37 pm
                  </span>
                </div>
              </div>
            </div>
            <div
              class="css-nen11g-MuiStack-root"
            >
              <div
                class="labelContainer"
                style="justify-content: unset;"
              >
                <span
                  style="margin-right: 8px; display: flex; align-items: center; width: 18px;"
                >
                  <svg
                    aria-hidden="false"
                    aria-label="Mail icon"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                    data-testid="ContactMailIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M21 8V7l-3 2-3-2v1l3 2 3-2zm1-5H2C.9 3 0 3.9 0 5v14c0 1.1.9 2 2 2h20c1.1 0 1.99-.9 1.99-2L24 5c0-1.1-.9-2-2-2zM8 6c1.66 0 3 1.34 3 3s-1.34 3-3 3-3-1.34-3-3 1.34-3 3-3zm6 12H2v-1c0-2 4-3.1 6-3.1s6 1.1 6 3.1v1zm8-6h-8V6h8v6z"
                    />
                  </svg>
                </span>
                <div
                  class="label"
                  style="padding-left: 4px; color: rgb(0, 59, 74);"
                >
                  Insurance
                </div>
              </div>
              <div
                class="css-1okj3ks-MuiStack-root"
              >
                <div
                  style="margin-top: 10px;"
                >
                  <span
                    class="MuiTypography-root MuiTypography-bodyMedium css-1oitrp4-MuiTypography-root"
                  >
                    No Insurance provided
                  </span>
                </div>
              </div>
            </div>
            <div
              class="css-nen11g-MuiStack-root"
            >
              <div
                class="labelContainer"
                style="justify-content: unset;"
              >
                <span
                  style="margin-right: 8px; display: flex; align-items: center; width: 18px;"
                >
                  <svg
                    aria-hidden="false"
                    aria-label="Business icon"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                    data-testid="BusinessIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M12 7V3H2v18h20V7H12zM6 19H4v-2h2v2zm0-4H4v-2h2v2zm0-4H4V9h2v2zm0-4H4V5h2v2zm4 12H8v-2h2v2zm0-4H8v-2h2v2zm0-4H8V9h2v2zm0-4H8V5h2v2zm10 12h-8v-2h2v-2h-2v-2h2v-2h-2V9h8v10zm-2-8h-2v2h2v-2zm0 4h-2v2h2v-2z"
                    />
                  </svg>
                </span>
                <div
                  class="label"
                  style="padding-left: 4px; color: rgb(0, 59, 74);"
                >
                  Purpose of visit
                </div>
              </div>
              <div
                class="css-1okj3ks-MuiStack-root"
              >
                <div
                  style="margin-top: 10px;"
                >
                  <span
                    class="MuiTypography-root MuiTypography-bodyMedium css-1oitrp4-MuiTypography-root"
                  >
                    -
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
